components/ServiceDetails/index.vue

```diff
@@ -14,6 +14,12 @@
         <el-row>
           <el-col :xs="24" :sm="24" :md="17" :lg="17" :xl="17">
             <el-tabs class="main" v-model="activeDescription" @tab-click="handleTabClick">
+              <el-tab-pane label="VERSIONS" name="versions">
```

Comment:
feels weird to have the versions first, i would put the doc + events/taskes first

Reply:
@NicolasMahe what do you think about this? I agree with @antho1404 on this one